w3-filecoin.md

```diff
@@ -363,6 +365,96 @@ If the added piece is invalid, the reason for the failure is also reported:
 }
 ```
 
+#### `filecoin/info`
+
+An agent MAY invoke the `filecoin/info` capability to request known information about content piece in Filecoin. See [schema](#filecoininfo-schema).
+
+> `did:key:zAliceAgent` invokes `filecoin/info` capability provided by `did:web:web3.storage`
+
+```json
+{
+  "iss": "did:key:zAliceAgent",
+  "aud": "did:web:web3.storage",
+  "att": [
+    {
+      "with": "did:key:zAlice",
```

Comment:
In the client we're using the space DID.

Reply:
this is what it was meaning to reflect here. We have `"did:key:zAliceAgent"` as the agent and `"did:key:zAlice"` as the space DID across all Storefront capabilities.

Should I add Space suffix to the resource of all `filecoin/*` to be more clear?

Reply:
Oh lol ok!